src/wallet/rpcwallet.cpp

```diff
@@ -2709,7 +2709,7 @@ static UniValue listunspent(const JSONRPCRequest& request)
             "    \"scriptPubKey\" : \"key\",   (string) the script key\n"
             "    \"amount\" : x.xxx,         (numeric) the transaction output amount in " + CURRENCY_UNIT + "\n"
             "    \"confirmations\" : n,      (numeric) The number of confirmations\n"
-            "    \"redeemScript\" : n        (string) The redeemScript if scriptPubKey is P2SH\n"
+            "    \"redeemScript\" : n        (string) The redeemScript if scriptPubKey is P2SH, or witnessScript if the scriptPubKey is P2WSH or P2SH-P2WSH\n"
```

Comment:
Why not a separate `witnessScript` field like in #14454?

Reply:
@Sjors because this has to be compatible with how signrawtransaction works, so it can be passed directly in. signrawtransaction only accepts the witness script and automatically wraps it

Reply:
Would it be super involved to add support for witnessScript to `signrawtransactionwithkey`?

Reply:
Agreed with @Sjors I'd much rather we fix terminology instead of compounding confusion.